tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/config/retry/ConfigPropertyGlobalVsClassVsMethodTest.java

```diff
@@ -65,7 +71,21 @@ void propertyPriorityTest() {
         catch (IllegalStateException e) {
             e.printStackTrace();
         }
-        Assert.assertEquals(bean.getRetry(), 7);
+        Assert.assertEquals(bean.getRetry(), 8);
+    }
+
+    /**
+     * Test that config property on file has precedence over annotation config
+     */
+    @Test
+    void propertyRetryOnPriorityTest() {
+        try {
+            bean.doProcessWithCustomException();
+        }
+        catch (CustomRetryServiceException e) {
+            Assert.assertTrue(e instanceof Exception);
```

Comment:
I think what you really want here is an empty `catch` block, but a `fail()` call in the `try` block, after `bean.doProcessWithCustomException()`.